lib/internal/bootstrap/node.js

```diff
@@ -49,6 +49,12 @@ setupProcessObject();
 setupGlobalProxy();
 setupBuffer();
 
+Object.defineProperty(process, Symbol.toStringTag, {
+  enumerable: false,
+  writable: false,
+  configurable: false,
+  value: 'process'
+});
 process.domain = null;
```

Comment:
👆 Should this be done in the `setupProcessObject()` helper?

Reply:
Good catch, I've added with a final CI run here.